app-editors/vscode/metadata.xml

```diff
@@ -13,6 +13,10 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
+	<use>
+		<flag name="wayland-on-wayland">Run in wayland mode in wayland desktops (xwayland otherwise)</flag>
```

Comment:
We talked over that in IRC, let's use `wayland` as the name.

Reply:
Done